packages/sfui/frameworks/nuxt/src/module.ts

```diff
@@ -0,0 +1,65 @@
+import { defineNuxtModule, addComponent, addImportsSources, installModule } from '@nuxt/kit';
+import * as storefrontUi from '@storefront-ui/vue';
+import { tailwindConfig } from '@storefront-ui/vue/tailwind-config';
+import { NuxtConfig } from '@nuxt/schema';
+// Module options TypeScript interface definition
+export interface ModuleOptions {
+  contentPath?: string;
+}
+
+export default defineNuxtModule<ModuleOptions>({
+  meta: {
+    name: 'storefront-ui',
+    configKey: 'storefrontUi',
+    compatibility: {
+      nuxt: '>=3.0.0',
+    },
+  },
+  // Default configuration options of the Nuxt module
+  defaults: {
+    contentPath: './node_modules/@storefront-ui/vue/**/*.mjs',
+  },
+  async setup(options, nuxt) {
+    const { contentPath } = options;
+
+    const nuxtOptions = nuxt.options as unknown as NuxtConfig;
```

Comment:
I cannot believe that this typing does not work out of the box 😄 but 🤷

Reply:
Tracked down the line causing issues to

```
declare module '@nuxt/schema' {
  interface NuxtConfig { ['tailwindcss']?: Partial<ModuleOptions> }
  interface NuxtOptions { ['tailwindcss']?: ModuleOptions }
}
```

`tailwindcss` uses NuxtOptions type without the Partial, because I think it's assuming this type would only be used by the tailwind module itself. 

I don't think nuxt/kit plays well with Typescript when one module is meant to influence the config of another.

Reply:
Nice investigation! As long as it's not on our side - let's just stick with working code